README.md

```diff
@@ -75,23 +72,44 @@ git checkout stable
 make install
 ```
 
-## Install for Development
+# Quick Start
+Users of PyGridSim have the option between creating a fully customized circuit and using PyGridSim-provided parameters to build their circuit. Consider the simplest circuit: one source, one load, and a line connecting them. The following code snippet demonstrates how to model and print results on this circuit on PyGridSim with both methods.
+
+## Customized Circuit Creation
+```python
+circuit = PyGridSim()
+
+# Add Custom Source and Load
+circuit.add_load_nodes(params={"kV": 0.12, "kW": 1, "kvar": 1})
+circuit.update_source(params={"kV": 0.5})
 
-If you want to contribute to the project, a few more steps are required to make the project ready
-for development.
+# Add Line
+circuit.add_lines([("source", "load0")], params={"length": 1})
 
-Please head to the [Contributing Guide](https://amzhao.github.io/PyGridSim/contributing.html#get-started)
-for more details about this process.
+# Solve and Print Results
+circuit.solve()
+print(circuit.results(["Voltages", "Losses"]))
+circuit.clear()
+```
```

Comment:
can you add a line after the code block to show the printed result with a few sentences to describe where is came from?